src/Chainweb/Rosetta/Util.hs

```diff
@@ -0,0 +1,686 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE BangPatterns #-}
+
+-- |
+-- Module: Chainweb.Rosetta.Util
+-- Copyright: Copyright © 2018 - 2020 Kadena LLC.
+-- License: MIT
+-- Maintainer: Linda Ortega <linda@kadena.io>
+-- Stability: experimental
+--
+--
+module Chainweb.Rosetta.Util where
```

Comment:
This isn't really `Util` as this is where "the sausage is made" as opposed to some grab-bag of utilities. `Internal` is probably more illustrative, although it could also be `Impl` or just `Rosetta`

Reply:
Moved block functions and tx matching functions to `Chainweb.Rosetta.Internal`.